src/main/java/com/example/WorldOfAnimals/services/AnimalBreedService.java

```diff
@@ -21,6 +28,30 @@ public class AnimalBreedService {
     private AnimalBreedRepository repository;
     private AnimalBreedMapper mapper;
 
+    @Transactional
+    public void saveBreeds() {
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
No need to create RestTemplate every time

1. Create folder config 
2. Create Base class AppConfiguratuion with anotation @Configuration 
3. Create bean RestTemplate by resource "https://dog.ceo/api/breeds/list/all"